subtitles/whisper_transcriber.py

```diff
@@ -21,7 +24,11 @@ def generate(self, source: str, language: str = None) -> (str, str):
         options['no_speech_threshold'] = 0.275
         options['logprob_threshold'] = None
 
-        result = self.__model.transcribe(source, **options, verbose=False)
+        try:
+            mutex_whisper.acquire()
+            result = self.__model.transcribe(source, **options, verbose=False)
+        finally:
+            mutex_whisper.release()
```

Comment:
I think that manually acquiring/releasing resources is likely error prawn. What do you think about this?
```suggestion
        with mutex_whisper:
            result = self.__model.transcribe(source, **options, verbose=False)
```